### PR TITLE
Integration test only for pr validation

### PR DESCRIPTION
**Pull request by amitkrout**
**What type of PR is this?**
/kind cleanup

**What does does this PR do / why we need it**:
Currently we have high volume of test running for each pr which consumes almost 3hrs. So the plan here is to distribute the test in different types run. For example in nightly run all the test script will run periodically on master branch over a period of time. The pr test will be validated against integration test only.

**Which issue(s) this PR fixes**:

Fixes - part of #2931

**How to test changes / Special notes to the reviewer**:
Each pr will be validated against integration test only on OpenShift CI

### CODE REVIEW

**Comment by amitkrout** [comment]
Once this pr goes in, i will update https://github.com/openshift/release/pull/8707 to make this changes reflected in OpenShift CI.

File: scripts/openshiftci-presubmit-integration-tests.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+# fail if some commands fails
+set -e
+# show commands
+set -x
+
+export CI="openshift"
+make configure-installer-tests-cluster
+make bin
+mkdir -p $GOPATH/bin
```
Comment (mohammedzee1000) [inline scripts/openshiftci-presubmit-integration-tests.sh:11]: should the `make bin` not happen after the bin directory is created?

Comment (prietyc123) [reply scripts/openshiftci-presubmit-integration-tests.sh:11]: It does not matter.
Actually the ```$GOPATH/bin``` is created to keep the ginkgo binary within the bin dir. ```make bin``` just create the odo binary in the odo clone dir.

**Comment by prietyc123** [comment]
Good to see separate integration test for only pr validation.

/lgtm

**Pull request merged**